input/resources/valueset-au-jurisdiction-extended.xml

```diff
@@ -1,28 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ValueSet xmlns="http://hl7.org/fhir">
-    <id value="rsg-source-document-jurisdiction" />
+    <id value="au-jurisdiction-extended" />
     <meta>
         <profile value="http://hl7.org/fhir/StructureDefinition/shareablevalueset" />
     </meta>
     <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm">
         <valueInteger value="0"/>
     </extension>
-    <url value="http://terminology.hl7.org.au/ValueSet/rsg-source-document-jurisdiction" />
+    <url value="http://terminology.hl7.org.au/ValueSet/jurisdiction-extended" />
     <identifier>
         <system value="urn:ietf:rfc:3986"/>
         <value value="urn:oid:2.16.840.1.113883.2.3.4.2.4.33"/>
     </identifier>
-    <name value="AURecordedSexOrGenderSourceDocumentType" />
-    <title value="AU Recorded Sex or Gender (RSG) Source Document Jurisdiction" />
+    <name value="JurisdictionAUExtended"/>
```

Comment:
Put the word ValueSet back in to the name and title.

Reply:
Just a query as to why we need the word ValueSet in the name and title when there are other similar examples where this is not the case. Also, it is easier to put the words value set after the link.
ServiceDeliveryLocationRoleTypeAUExtended, 
CoverageTypeAndSelfPayCodesAUExtended,
ActEncounterCodeAUExtended. (this is the example in the Wiki)
e.g.
shall be a member of the Jurisdiction - AU Extended value set

Reply:
We need it because it matches the conventions. The convention is to take the International title and name and add "AU Extended". The international title and name contain the word "ValueSet".

Reply:
Updated names/titles as requested.